### PR TITLE
config/testgrids/config.yaml: add new jobs for OpenShift release-blocking dashboard

### PR DESCRIPTION
**Pull request by abhinavdahiya**
Adds azure-4.2, azure-serial-4.2, gcp-4.2, gcp-serial-4.2 jobs for release-blocking.

/cc @smarterclayton @stevekuznetsov

### CODE REVIEW

**Comment by abhinavdahiya** [comment]
Requires https://github.com/openshift/release/pull/4698

File: config/testgrids/config.yaml [hunk 1]
```diff
@@ -619,6 +619,14 @@ test_groups:
   gcs_prefix: origin-ci-test/logs/release-openshift-ocp-installer-e2e-aws-4.2
 - name: release-openshift-ocp-installer-e2e-aws-serial-4.2
   gcs_prefix: origin-ci-test/logs/release-openshift-ocp-installer-e2e-aws-serial-4.2
+- name: release-openshift-ocp-installer-e2e-azure-4.2
```
Comment (chases2) [inline config/testgrids/config.yaml]: Please put these additions (as well as any other openshift tests you want to include) in a separate file

```
test_groups:
 <all this stuff>
```

File: config/testgrids/config.yaml [hunk 2]
```diff
@@ -2518,6 +2526,22 @@ dashboards:
     description: Runs Kubernetes serial e2e tests against an OpenShift 4.2 OCP cluster.
     test_group_name: release-openshift-ocp-installer-e2e-aws-serial-4.2
     base_options: width=10
+  - name: redhat-release-openshift-ocp-installer-e2e-azure-4.2
+    description: Runs Kubernetes e2e tests against an OpenShift 4.2 OCP cluster.
```
Comment (chases2) [inline config/testgrids/config.yaml]: Similarly, you can move an entire dashboard

```
dashboards:
- name: redhat-openshift-release-blocking
  dashboard_tab:
  ...
```

**Pull request closed without merging**